chia/cmds/farm_funcs.py

```diff
@@ -181,69 +181,101 @@ async def challenges(farmer_rpc_port: int, limit: int) -> None:
         )
 
 
-async def summary(rpc_port: int, wallet_rpc_port: int, harvester_rpc_port: int, farmer_rpc_port: int) -> None:
+class Summary:
+    def __init__(self):
+        self.status = None
+        self.total_chia_farmed = None
+        self.user_transaction_fees = None
+        self.block_rewards = None
+        self.last_height_farmed = None
+        self.total_plot_size = None
+        self.plot_count = None
+        self.estimated_network_space = None
+        self.expected_time_to_win = None
+
+    def __str__(self):
+        def format(name: str, value: str) -> str:
```

Comment:
I note that it doesn't seem like it's expected that the inputs are actually all string.